ui/src/test/java/pages/InvestorsPage.java

```diff
@@ -0,0 +1,39 @@
+package pages;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+import com.codeborne.selenide.WebDriverRunner;
+import io.cucumber.datatable.DataTable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.codeborne.selenide.Selenide.$$x;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class InvestorsPage {
+
+    String listOfLinksActualXpath = ".//section//ul[contains(@class,'list')]/*";
+    SelenideElement listOfLinksActualElement = $x(listOfLinksActualXpath);
+    ElementsCollection listOfLinksActualCollection = $$x(listOfLinksActualXpath);
+
+    public boolean isUrlCorrect(String url) {
```

Comment:
this method is common for all pages, so good practice to create class like CommonPage (could be abstract) move all such methods in this class and extend all concrete pages form this class

Reply:
The MainPage left empty... I'm not sure this is OK.